Libraries/NativeAnimation/Nodes/RCTPropsAnimatedNode.m

```diff
@@ -54,29 +77,30 @@ - (void)performUpdate
 {
   [super performUpdate];
 
+  // Since we are updating nodes after detaching them from views there is a time where it's
+  // possible that the view was disconnected and still receive an update, this is normal and we can
+  // simply skip that update.
   if (!_connectedViewTag) {
-    RCTLogError(@"Node has not been attached to a view");
     return;
   }
 
-  NSMutableDictionary *props = [NSMutableDictionary dictionary];
   [self.parentNodes enumerateKeysAndObjectsUsingBlock:^(NSNumber * _Nonnull parentTag, RCTAnimatedNode * _Nonnull parentNode, BOOL * _Nonnull stop) {
 
     if ([parentNode isKindOfClass:[RCTStyleAnimatedNode class]]) {
-      [props addEntriesFromDictionary:[(RCTStyleAnimatedNode *)parentNode propsDictionary]];
+      [_propsDictionary addEntriesFromDictionary:[(RCTStyleAnimatedNode *)parentNode propsDictionary]];
 
     } else if ([parentNode isKindOfClass:[RCTValueAnimatedNode class]]) {
       NSString *property = [self propertyNameForParentTag:parentTag];
       CGFloat value = [(RCTValueAnimatedNode *)parentNode value];
-      [props setObject:@(value) forKey:property];
+      [_propsDictionary setObject:@(value) forKey:property];
     }
 
   }];
 
```

Comment:
If there are props that were previously set, but now are no longer set -- that is, if the old value of `_propsDictionary` has keys that wouldn't be in `props` (in the old code) -- should we set those entries in `_propsDictionary` to `[NSNull null]`?